hydra-node/src/Hydra/Options.hs

```diff
@@ -352,21 +332,16 @@ instance FromJSON ChainConfig where
         tag -> fail $ "unexpected tag " <> tag
 
 data OfflineChainConfig = OfflineChainConfig
-  { initialUTxOFile :: FilePath
+  { offlineHeadSeed :: HeadSeed
```

Comment:
so, if i close an offline head then i need to prune persistency and restart the node with diff config?

for online heads this is not the case.
i would maintain consistency btw both ux.

wdyt?